README.md

```diff
@@ -132,6 +127,38 @@ CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bi
     | mistralai/Mistral-7B-v0.1 | 4.34/4.36 |
     
 
+### Deployment
+The quantized models support export to Int4 precision and can be run on Intel CPU using [neural-speed](examples/huggingface/neural_speed/runtime_acc.py), it also supports deployment on GPU using autogptq.
+
+After the quantization process is complete, you can invoke the export API to perform the conversion and save the model along with the quantization configuration file.
+```python
+  from transformers import AutoModelForCausalLM, AutoTokenizer
+  from auto_round import AutoRound
+
+  model = AutoModelForCausalLM.from_pretrained(
+            model_name, low_cpu_mem_usage=True, torch_dtype="auto", trust_remote_code=True
+        )
+  tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
+  autoround = AutoRound(model, tokenizer, bits=4, group_size=128, scheme="asym")
+  fake_qdq_model,weight_config = autoround.quantize() ##scale,zp info are saved in weight config dict
+
+  ## export for CPU deployment
+  from auto_round import  QuantConfig, compress_model, save_compressed_model
+  output_dir = "/PATH/TO/SAVE/COMPRESSED/MODEL/"
```

Comment:
how abouts just following export_to_autogpt,  have a new function export_to_neuralspeed or export_to_cpu, this will ease the usage